tiles info needs to be added to XML schema

```
[100%] Generating arch.merged.lint.html
...
github/SymbiFlow/symbiflow-arch-defs-v2x/build/xc7/archs/artix7/devices/xc7a50t-basys3-roi-virt/arch.unique_pack.xml:331: element tiles: Schemas validity error : Element 'tiles': This element is not expected.                                                         
```

This is also happening if you run the [`./.github/travis/script.sh`](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/.github/travis/script.sh) on current head (26687ddc283a255ad804453d1f61e8f71e6f533a).

```
tansell@tansell:~/github/SymbiFlow/symbiflow-arch-defs$ ./.github/travis/script.sh 
....
[ 50%] Built target file_xc7_archs_zynq7_tiles_slicem_slicem.pb_type.xml
[ 50%] Built target file_xc7_archs_zynq7_tiles_slicem_slicem.model.xml
[ 50%] Built target file_xc7_archs_zynq7_devices_xc7z010-zybo-roi-virt_arch.xml
[ 50%] Built target file_xc7_archs_zynq7_devices_xc7z010-zybo-roi-virt_arch.merged.xml
Scanning dependencies of target zynq7_xc7z010-zybo-roi-virt_arch_lint
[ 50%] Generating arch.merged.lint.html
/usr/local/google/home/tansell/github/SymbiFlow/symbiflow-arch-defs/build/xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/arch.unique_pack.xml:331: element tiles: Schemas validity error : Element 'tiles': This element is not expected.     
/usr/local/google/home/tansell/github/SymbiFlow/symbiflow-arch-defs/build/xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/arch.unique_pack.xml fails to validate                                                                               
xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/CMakeFiles/zynq7_xc7z010-zybo-roi-virt_arch_lint.dir/build.make:62: recipe for target 'xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/arch.merged.lint.html' failed                             
make[3]: *** [xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/arch.merged.lint.html] Error 3
make[3]: *** Deleting file 'xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/arch.merged.lint.html'
CMakeFiles/Makefile2:59508: recipe for target 'xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/CMakeFiles/zynq7_xc7z010-zybo-roi-virt_arch_lint.dir/all' failed                                                                                
make[2]: *** [xc7/archs/zynq7/devices/xc7z010-zybo-roi-virt/CMakeFiles/zynq7_xc7z010-zybo-roi-virt_arch_lint.dir/all] Error 2                                                                                                                
CMakeFiles/Makefile2:1732: recipe for target 'CMakeFiles/all_xml_lint.dir/rule' failed
make[1]: *** [CMakeFiles/all_xml_lint.dir/rule] Error 2
Makefile:760: recipe for target 'all_xml_lint' failed
make: *** [all_xml_lint] Error 2

```

--- a/common/xml/fpga_architecture.xsd
+++ b/common/xml/fpga_architecture.xsd
@@ -584,6 +584,41 @@
     <xs:attribute name="name" type="xs:string" use="required"/>
   </xs:complexType>
 
+  <xs:complexType name="map">
+    <xs:attribute name="from" type="xs:string" use="required"/>
+    <xs:attribute name="to" type="xs:string" use="required"/>
+    <xs:attribute name="num_pins" type="xs:string"/>
+  </xs:complexType>
+
+  <xs:complexType name="tile_mode">
+    <xs:choice maxOccurs="unbounded" minOccurs="0">
+      <xs:element name="map" type="map"/>
+    </xs:choice>
+    <xs:attribute name="name" type="xs:string" use="required"/>
+  </xs:complexType>
+
+  <xs:complexType name="equivalent_tiles">
+    <xs:choice maxOccurs="unbounded" minOccurs="0">
+      <xs:element name="mode" type="tile_mode"/>
+    </xs:choice>
+  </xs:complexType>
+
+  <xs:complexType name="tile">
+    <xs:choice maxOccurs="unbounded" minOccurs="0">
+      <xs:element name="equivalent_tiles" type="equivalent_tiles"/>
+      <xs:element name="pinlocations" type="pinlocations"/>
+      <xs:element name="fc" type="fc"/>
+      <xs:element name="switchblock_locations" type="switchblock_locations"/>
+    </xs:choice>
+    <xs:attribute name="name" type="xs:string" use="required"/>
+  </xs:complexType>
+
+  <xs:complexType name="tiles">
+    <xs:sequence>
+      <xs:element name="tile" type="tile" maxOccurs="unbounded"/>
+    </xs:sequence>
+  </xs:complexType>
+
   <xs:complexType name="pb_type">
     <xs:choice maxOccurs="unbounded" minOccurs="0">
       <xs:element name="pb_type" type="pb_type"/>
@@ -792,6 +827,7 @@
         <xs:element name="power" type="global_power"/>
         <xs:element name="clocks" type="clocks"/>
         <xs:element name="directlist" type="directlist"/>
+        <xs:element name="tiles" type="tiles"/>
         <xs:element name="complexblocklist" type="complexblocklist"/>
       </xs:choice>
     </xs:complexType>

